src/Microsoft.AspNet.Mvc.Core/SessionStateTempDataProvider.cs

```diff
@@ -16,8 +17,8 @@ namespace Microsoft.AspNet.Mvc
     /// </summary>
     public class SessionStateTempDataProvider : ITempDataProvider
     {
-        private static JsonSerializer jsonSerializer = new JsonSerializer();
-        private static string TempDataSessionStateKey = "__ControllerTempData";
+        private string TempDataSessionStateKey = "__ControllerTempData";
```

Comment:
Actually @kichalla suggested that since this is a Singleton service, there is no point in having them as static.